lib/rails_admin/extensions/paper_trail/auditing_adapter.rb

```diff
@@ -116,7 +116,7 @@ def listing_for_model_or_object(model, object, query, sort, sort_reverse, all, p
 
           current_page = page.presence || '1'
 
-          versions = object.nil? ? versions_for_model(model) : object.versions
+          versions = object.nil? ? versions_for_model(model) : object.send(model.model.versions_association_name)
```

Comment:
I believe `.versions_association_name` is public

```suggestion
          versions = object.nil? ? versions_for_model(model) : object.public_send(model.model.versions_association_name)
```

Also is this covered by any existing tests?

Reply:
Sidenote - this solves the same problem as https://github.com/sferik/rails_admin/pull/3269 with a _slightly_ different approach (IIRC `object.version_association_name` will execute a SQL query, while calling from the model does not)

Reply:
It is public indeed, I'll change it to public_send. I used send to keep it in line with the other `send` usage in the file.

There was no test covering the versions usage, only one regarding the user used for the audit.